draft-ietf-quic-transport.md

```diff
@@ -1584,7 +1584,8 @@ of Initial packets that it sent. Including connection ID values in transport
 parameters and verifying them ensures that that an attacker cannot influence
 the choice of connection ID for a successful connection by injecting packets
 carrying attacker-chosen connection IDs during the handshake. An endpoint MUST
-treat any of the following as a connection error of type PROTOCOL_VIOLATION:
+treat any of the following as a connection error of type
+TRANSPORT_PARAMETER_ERROR:
```

Comment:
A list is good here, but these aren't all equal.  In particular, the behaviour that is conditioned on whether a Retry was sent is not consistent with your principle.  I would argue that those are also PROTOCOL_VIOLATION cases.

If you agree, then you can move those two into the next list (which currently only has one item).  That leaves two items in this list and I think that can be turned back into prose.

Reply:
Agreed. And fixed.

Reply:
I made a prose suggestion in a PR, which you can take or leave.

Reply:
Thank you. Merged.